src/seedu/addressbook/common/Messages.java

```diff
@@ -6,8 +6,9 @@
 public class Messages {
 
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
-    public static final String MESSAGE_PERSON_NOT_IN_ADDRESSBOOK = "Person could not be found in address book";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid."
+            + "Run the list command to view all persons.";
```

Comment:
Maybe add the findcommand too then, to suggest to user on how to resolve

Reply:
Issue is if we're gonna add more commands that generates a list of person. Don't think we are though...
Hmm I guess it can read "Run the list/find command to generate a list of persons"